internal/character/natasha/technique.go

```diff
@@ -0,0 +1,43 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	technique                key.Modifier = "natasha-technique"
+	techniqueScalePercentage float64      = 0.80
+)
+
+func init() {
+	modifier.Register(technique, modifier.Config{
+		StatusType: model.StatusType_STATUS_DEBUFF,
+		Duration:   1,
+		TickMoment: modifier.ModifierPhase1End,
+	})
```

Comment:
* Needs BehaviorFlag `STAT_FATIGUE`
* Stacking: `ReplaceBySource`
* `TickMoment` should be unset/left at default of `ModifierPhase2End`